package.json

```
@@ -3,7 +3,7 @@
     "displayName": "Calva: Clojure & ClojureScript Interactive Programming",
     "description": "Integrated REPL, formatter, Paredit, and more. Powered by cider-nrepl and clojure-lsp.",
     "icon": "assets/calva.png",
-    "version": "2.0.185",
+    "version": "2.0.185-monorepo-proj-root-dir",
```

```suggestion
    "version": "2.0.185",
```
(I usually pick up the VSIX from CircleCI instead. Maybe we should make a script that build a local VSIX...)